### PR TITLE
Add cudaEventBlockingSync to event creation

### PR DESCRIPTION
**Pull request by DerWaldschrat**

### CODE REVIEW

**Comment by ax3l** [comment]
@DerWaldschrat CI seems to indicate you need to add CPU backend handling

**Comment by psychocoderHPC** [comment]
grrrrr the interfaces for CPU and GPU backends to create an event are different. :-(

File: include/cupla/manager/Event.hpp [hunk 1]
```diff
@@ -61,7 +61,8 @@ namespace detail
             hasTimer( !( flags & cuplaEventDisableTiming ) ),
             event(
                 new AlpakaEvent(
-                    Device< T_DeviceType >::get().current()
+                    Device< T_DeviceType >::get().current(),
+                    !(flags & cuplaEventBlockingSync)
```
Comment (psychocoderHPC) [inline include/cupla/manager/Event.hpp]: please add here `#if( ALPAKA_ACC_GPU_HIP_ENABLED == 1 || ALPAKA_ACC_GPU_CUDA_ENABLED == 1 )` to handle the interface differences in alpaka.

Comment (DerWaldschrat) [reply include/cupla/manager/Event.hpp]: Should be fixed now.

**Comment by ax3l** [comment]
> grrrrr the interfaces for CPU and GPU backends to create an event are different. :-(

@DerWaldschrat can you please open an upstream alpaka issue for this?

**Pull request merged**